platform/platform-filter-response/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -25,10 +26,13 @@
         <AD description="The HTTP X-XSS-Protection response header is a feature that stops pages from loading when they detect reflected cross-site scripting (XSS) attacks."
             name="X-XSS-Protection" id="xXssProtection" required="true" type="String"
             default="1; mode=block"/>
+        <AD description="The Cache-Control header specifies the behavior for browsers and proxies to cache web resources."
```

Comment:
I debated that (and can still make that change if necessary). It is a bit weird because this is not the only filter that adds header values. Would we collapse everything into one list configuration? Only some things? Just this plugin? I'm also not aware of any compelling reasons for being able to add arbitrary header values, but could easily be persuaded otherwise. It just doesn't seem like the kind of thing an admin should be doing, or if they really want to add arbitrary header values, they're probably capable of writing and deploying their own filter.